### PR TITLE
#28 report open frames

### PR DESCRIPTION
**Pull request by Douglasdc3**
Implements #28 and adds test

### CODE REVIEW

File: tests/Feature/ReportCommandTest.php [hunk 1]
```diff
@@ -60,17 +60,17 @@ public function testReport()
 
     public function testReportWithProjectFilter()
     {
-        $project = factory(project::class)->create(['name' => 'blog']);
```
Comment (matt-allan) [inline tests/Feature/ReportCommandTest.php]: Thanks, not sure how those got lowercased.

**Comment by Douglasdc3** [comment]
Yep forgot to run style fix should be ok now.

**Comment by Douglasdc3** [comment]
Style fix done

**Comment by matt-allan** [comment]
Thanks!

**Pull request merged**